public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/SimpleCondition.tsx

```diff
@@ -71,24 +73,37 @@ export const SimpleConditionEditor = ({
     updateThresholdFunction(value.value ?? EvalFunction.IsAbove, expressionQueriesList, dispatch);
   };
 
-  const onEvaluateValueChange = (event: FormEvent<HTMLInputElement>, index?: number) => {
-    if (isRange) {
+  const onEvaluateValueChange = (event: FormEvent<HTMLInputElement>, index = 0) => {
+    const value = event.currentTarget.value;
+
+    // Allow empty value
+    if (value === '') {
       const newParams = produce(simpleCondition.evaluator.params, (draft) => {
-        draft[index ?? 0] = parseFloat(event.currentTarget.value);
+        draft[index ?? 0] = undefined;
```

Comment:
`index` defaults to `0` so it should never be undefined

Reply:
💯